hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -243,7 +243,7 @@
 
   public static final ConfigProperty<Boolean> PRESERVE_COMMIT_METADATA = ConfigProperty
       .key("hoodie.compaction.preserve.commit.metadata")
-      .defaultValue(false)
+      .defaultValue(true)
       .sinceVersion("0.11.0")
       .withDocumentation("When rewriting data, preserves existing hoodie_commit_time");
```

Comment:
```suggestion
      .withDocumentation("When running compaction, preserves existing `_hoodie_commit_time`");
```

a side question: this won't affect clustering, will it?

Reply:
yes, we have a separate config for clustering.